backend/data/db.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I have no idea where this file came from lol this is from my original push weeks ago!